cmd/config.go

```diff
@@ -31,16 +31,23 @@ func (d *DeisCmd) ConfigList(appID string, oneLine bool) error {
 
 	keys := sortKeys(config.Values)
 
-	if oneLine {
+	var configOutput *bytes.Buffer = new(bytes.Buffer)
```

Comment:
Just initializing the buffer as per the docs. Is there some better way to initialize a bytes.buffer?